### ISSUE TITLE
This does not compile on Windows no more.

### ISSUE THREAD

**Issue by stela2502** [COMBINE-lab/kmers#15]
Hi Team!
Great software and I have a lot of fun with it! But after my last ```cargo clean``` I am not longer able to compile ```kmers``` on Windows11. I know most bioinformaticians are not working on Windows and nether am I, BUT we also have a Windows specific VR application that I use quite a lot (CellexalVR). We developed that our selves and it is really cool to work with single-cell data in that tool. Just if you want to try. Anyhow I therefore work on Windows and it would be really really cool if I could exclude this "simple-sds" lib from "my" kmers ;-)

Meaning could you PLEASE add a key somewhere before you included simple-sds (2 weeks ago) so that I can depend on kmers before this lib has been added? That would be very helpful! I do not assume simple-sds will support windows in the short term - or?

Thank you!

**Comment by theJasonFan** [comment]
Yes, I'll make SeqVector a feature and update this on the dev branch. I don't have a windows machine to test this on right now. Once I make a change, @stela2502 would you mind seeing if it works for you on your OS?

**Comment by rob-p** [comment]
Hi @stela2502,

  I have merged @theJasonFan's PR into the main branch.  Pending us getting a windows CI up and working, would you mind testing now?  Basically, @theJasonFan has made the seq-vector module an optional dependency, so if you don't declare it when you pull the `kme.rs` crate in, then it shouldn't get compiled and everything should work again on Windows.

**Comment by stela2502** [comment]
I'll give it a try. And thank you for the quick response!

**Comment by stela2502** [comment]
Thank you - kmers does compile if I pull it from git, but if I try to compile it in my Rustody it breaks. I have removed all git copies of kmers that rust had cached and it even pulled your recent commit, but the error still persists. I do not know what else to try. You have no windows machine to test it on - or?
I have now nuked my repo and am going to clone again. Lets see if that changes anything.
